docker-compose.yml

```diff
@@ -128,12 +127,10 @@ services:
       - "8983:8983"
     image: adr.solr:${IMAGE_TAG}
     build:
-      context: .
+      context: ./
```

Comment:
let's just set the context here to `context: ./solr` so later in the Dockerfile you can simply use:

```
# Adding Files
ADD schema.xml /opt/solr/server/solr/configsets/ckan/conf/managed-schema
```

and remove `BUILD_CONTEXT` from the args below and user simple `dockerfile: Dockerfile`.

That's going to make solr docker-compose simillar to the rest of the services.